libs/payments/ui/src/lib/actions/updateCart.ts

```diff
@@ -23,4 +25,6 @@ export const updateCartAction = async (
       cartDetails,
     })
   );
+
+  revalidatePath('/');
```

Comment:
Why revalidate the root path? Shouldn't it be the path where the action is called from?